x-pack/plugins/siem/public/components/page/network/users_table/columns.tsx

```diff
@@ -37,104 +27,58 @@ export const getUsersColumns = (
     truncateText: false,
     hideForMobile: false,
     sortable: true,
-    render: userName => {
-      if (userName != null) {
-        const id = escapeDataProviderId(`${tableId}-table-${flowTarget}-user-${userName}`);
-        return (
-          <DraggableWrapper
-            key={id}
-            dataProvider={{
-              and: [],
-              enabled: true,
-              id,
-              name: userName,
-              excluded: false,
-              kqlQuery: '',
-              queryMatch: { field: 'user.name', value: userName },
-            }}
-            render={(dataProvider, _, snapshot) =>
-              snapshot.isDragging ? (
-                <DragEffects>
-                  <Provider dataProvider={dataProvider} />
-                </DragEffects>
-              ) : (
-                <>{userName}</>
-              )
-            }
-          />
-        );
-      } else {
-        return getEmptyTagValue();
-      }
-    },
+    render: userName =>
+      getRowItemDraggable({
+        rowItem: userName,
+        attrName: 'user.name',
+        idPrefix: `${tableId}-table-${flowTarget}-user-${userName}`,
```

Comment:
I think you can just supply `${tableId}-table-${flowTarget}` for `idPrefix` as `getRowItemDraggable()` will append the attrName + value.

In `getRowItemDraggable()`:
``` typescript 
const id = escapeDataProviderId(`${idPrefix}-${attrName}-${rowItem}`);
```

Reply:
🦅 👁 ! Thanks